osf_tests/test_remind_draft_preregistrations.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from django.utils import timezone
+from website import settings
+
+from .factories import UserFactory, DraftRegistrationFactory
+
+from osf.models import MetaSchema, DraftRegistrationApproval, QueuedMail
+from osf.models.queued_mail import PREREG_REMINDER_TYPE
+
+from scripts.remind_draft_preregistrations import main
+
+@pytest.fixture()
+def user():
+    return UserFactory(is_registered=True)
+
+@pytest.fixture()
+def schema():
+    return MetaSchema.objects.get(name='Prereg Challenge')
+
+@pytest.mark.django_db
+class TestPreregReminder:
+
+    @pytest.fixture()
+    def draft(self, user, schema):
+        draft = DraftRegistrationFactory(
+            registration_schema=schema, initiator=user
+        )
+        draft.datetime_initiated = timezone.now() - settings.PREREG_WAIT_TIME
+        draft.save()
+        return draft
+
+
+    def test_trigger_prereg_reminder(self, draft):
+        main(dry_run=False)
+
+        assert len(QueuedMail.objects.filter(email_type=PREREG_REMINDER_TYPE)) == 1
```

Comment:
Prefer to use count on querysets `QueuedMail.objects.filter(email_type=PREREG_REMINDER_TYPE).count() == 1`

> len(). A QuerySet is evaluated when you call len() on it. This, as you might expect, returns the length of the result list.
> 
> Note: If you only need to determine the number of records in the set (and don’t need the actual objects), it’s much more efficient to handle a count at the database level using SQL’s SELECT COUNT(*). Django provides a count() method for precisely this reason.